e2e/basic/manifests/pods.yml

```diff
@@ -0,0 +1,45 @@
+# Pods for basic eBPF agent test
+# client pod just sends periodic HTTP requests to an HTTP service (nginx)
+apiVersion: v1
+kind: Pod
+metadata:
+  name: client
+spec:
+  containers:
+    - name: fedora
+      image: fedora:36
+      command:
+        - sh
+        - -c
+        - "echo 'start pinging from client pod...'; while true; do curl -v http://server ; sleep 1; done"
```

Comment:
What about sending some data in curl command here ? 

So we will be able to compare bytes of flows we got
`curl -v -d 'SOME_JSON_DATA' -H "Content-Type: application/json" -X POST http://server`

Reply:
there is also `ping -s http://server` as an alternative